### ISSUE TITLE
If an auth token isn't accepted, restart the implicit login flow and try again

### ISSUE THREAD

**Issue by caitp** [caitp/TwitchSwitcher#14]
Right now, once an auth token is received, a new one won't be requested until OBS is restarted / the plugin is reloaded. I've pinged @TwitchDev about how long auth tokens are valid for, but have not heard a response from anyone yet. Presumably, they're not valid forever.

**Comment by Bootscreen** [comment]
hello,
any news about that? currently after every OBS restart i must relog me on the first scene change :(
maybe you can change it to the way like chatty or the witch bots work? so we can authorize OBS with a oauth token.

**Comment by caitp** [comment]
I agree it's pretty terrible at the moment :( I'll see if I can work on this on the weekend.

**Comment by Bootscreen** [comment]
hello again,
any news about it?

**Comment by caitp** [comment]
have not had much time to work on this lately, or to stream at all.

Contributions are welcome if you'd like to submit a fix, or know someone who might
